lib/datasource/go/index.ts

```diff
@@ -62,16 +73,22 @@ async function getDatasource(goModule: string): Promise<DataSource | null> {
  *
  * This function will:
  *  - Determine the source URL for the module
- *  - Call the respective getReleases in github to retrieve the tags
+ *  - Call the respective getReleases in github/gitlab to retrieve the tags
  *  - Filter module tags according to the module path
  */
 export async function getReleases({
   lookupName,
 }: GetReleasesConfig): Promise<ReleaseResult | null> {
   logger.trace(`go.getReleases(${lookupName})`);
   const source = await getDatasource(lookupName);
-  if (source && source.datasource === github.id) {
-    const res = await github.getReleases(source);
+  if (
+    source &&
+    (source.datasource === github.id || source.datasource === gitlab.id)
+  ) {
```

Comment:
Please invert the condition and return null early.

Reply:
Can be simplified to: `source?.datasource !== github.id && source?.datasource !== gitlab.id`